Assignment/Assignment.Tests/PingProcessTests.cs

```diff
@@ -58,65 +59,90 @@ public void Run_CaptureStdOutput_Success()
     public void RunTaskAsync_Success()
     {
         // Do NOT use async/await in this test.
-        // Test Sut.RunTaskAsync("localhost");
+        AssertValidPingOutput(Sut.RunTaskAsync("localhost").Result);
     }
 
+    //Review later if needed.
+    // [TestMethod]
+
+    // public void RunTaskAsync_Fail()
+    // {
+    //     // Do NOT use async/await in this test.
+    //     AssertValidPingOutput(Sut.RunTaskAsync("qqqqqqqqqqqqqq").Result);
+    // }
+
     [TestMethod]
     public void RunAsync_UsingTaskReturn_Success()
     {
         // Do NOT use async/await in this test.
-        PingResult result = default;
-        // Test Sut.RunAsync("localhost");
+        Task<PingResult> task = Sut.RunAsync("localhost");
+        PingResult result = task.Result;
         AssertValidPingOutput(result);
     }
 
     [TestMethod]
-#pragma warning disable CS1998 // Remove this
     async public Task RunAsync_UsingTpl_Success()
     {
         // DO use async/await in this test.
-        PingResult result = default;
-
-        // Test Sut.RunAsync("localhost");
+        Task<PingResult> task = Sut.RunAsync("localhost");
+        PingResult result = await task;

```

Comment:
You would typically do these on the same line:
```suggestion
        PingResult task = await Sut.RunAsync("localhost");
```